lib/wallet/rpc.js

```diff
@@ -1742,26 +1742,52 @@ class RPC extends RPCBase {
   }
 
   async getBids(args, help) {
-    if (help || args.length > 2)
-      throw new RPCError(errs.MISC_ERROR, 'getbids "name"  ( own )');
+    if (help || args.length > 3)
```

Comment:
I'd ask for braces for if clauses everywhere.

Reply:
Throughout codebase there's a single rule around braces, single liners w/o braces.
But if something takes more then one line, then you add braces. (p.s. it can be single command but broken into multiple lines, like in `throw new RPCError(\nerrs....`

Reply:
oh, this is multiline throw, we need braces.

Reply:
👍